daemon/logger/jsonfilelog/read.go

```diff
@@ -171,9 +178,21 @@ func followLogs(f *os.File, logWatcher *logger.LogWatcher, notifyRotate chan int
 	}
 	defer func() {
 		f.Close()
+		fileWatcher.Remove(name)
 		fileWatcher.Close()
 	}()
 
+	logWatcherCtx, logWatcherClosed := context.WithCancel(ctx)
```

Comment:
I think top context is unnecessary(maybe we'll pass it from api call later).
So, I think this line should look like:
```
ctx, cancel := context.WithCancel(ctx)
defer cancel()
```